releases/release-5.0.3.md

```diff
@@ -0,0 +1,114 @@
+    - Support push down broadcast cartesian join to TiFlash [#25106](https://github.com/pingcap/tidb/pull/25106)
+    - Planner/core: support union all for mpp. [#25051](https://github.com/pingcap/tidb/pull/25051)
+    - Balance region for batch cop task [#24724](https://github.com/pingcap/tidb/pull/24724)
+    - Store/copr: invalidate stale regions for Mpp query. [#24432](https://github.com/pingcap/tidb/pull/24432)
+
++ TiFlash
+
+    - Support casting the `STRING`  type to `DOUBLE` type
+    - Support `STR_TO_DATE` function
+    - Optimize non-joined data in right outer join by multi-threads
+    - Support cartesian join
+    - Support `LEFT` and `RIGHT` function
+    - Invalidate stale regions for MPP queries
+    - Support `ABS` function
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix the issue that an incorrect result is returned when using merge join on the `SET` types. [#25694](https://github.com/pingcap/tidb/pull/25694)
+    - Fix the `IN` expression arguments corruption issue. [#25666](https://github.com/pingcap/tidb/pull/25666)
+    - Avoid sessions of GC being affected by global variables [#25609](https://github.com/pingcap/tidb/pull/25609)
+    - Make sure limit outputs no more columns than its child [#25517](https://github.com/pingcap/tidb/pull/25517)
+    - Planner: fix a panic caused by sinking a Limit with inlined Proj into IndexLookUp when accessing a partition table [#25139](https://github.com/pingcap/tidb/pull/25139)
+    - Executor: fix ifnull bug when arg is enum/set [#25116](https://github.com/pingcap/tidb/pull/25116)
+    - Generate correct number of rows when all agg funcs are pruned [#25062](https://github.com/pingcap/tidb/pull/25062)
+    - Executor: make the ParallelApply be safe to be called again after returning empty results [#25011](https://github.com/pingcap/tidb/pull/25011)
+    - Fix the case that there could be duplicate ranges for multi-column index. [#24635](https://github.com/pingcap/tidb/pull/24635)
+    - Fix a nulleq bug [#24633](https://github.com/pingcap/tidb/pull/24633)
+    - Executor: fix data race of parallel apply operator [#24345](https://github.com/pingcap/tidb/pull/24345)
+    - Fix an issue that sorting on index-merge results in partition union reports 'index out of range'. [#24155](https://github.com/pingcap/tidb/pull/24155)
+    - Do not allow setting read timestamp to a future time. [#25761](https://github.com/pingcap/tidb/pull/25761)
+    - Fix the issue that ODBC-styled literal(like `{d '2020-01-01'}`...) cannot be used as the expression. [#25577](https://github.com/pingcap/tidb/pull/25577)
+    - Planner: select distinct should bypass batchget [#25533](https://github.com/pingcap/tidb/pull/25533)
+    - Store/tikv: change backoff type for missed tiflash peer. [#24600](https://github.com/pingcap/tidb/pull/24600)
+    - Fix index-out-of-range error when checking only_full_group_by [#24016](https://github.com/pingcap/tidb/pull/24016)
+
```

Comment:
```suggestion
+ TiKV

    - Fix wrong tikv_raftstore_hibernated_peer_state metric [#10431](https://github.com/tikv/tikv/pull/10431)
    - Copr: fix the wrong arguments type of json_unquote [#10424](https://github.com/tikv/tikv/pull/10424)
    - Fix the issue that br reports file already exists error when TDE enabled during restoration [#10421](https://github.com/tikv/tikv/pull/10421)
    - Skip clearing callback during graceful shutdown to avoid breaking ACID in some cases [#10396](https://github.com/tikv/tikv/pull/10396)
    - Fix the bug that share read index for replica reads on a leader [#10391](https://github.com/tikv/tikv/pull/10391)
    - Copr: fix wrong function cast double to double [#10388](https://github.com/tikv/tikv/pull/10388)
```